compiler/circle-mpqsolver/src/core/Quantizer.h

```diff
@@ -48,8 +48,8 @@ class Quantizer
   {
     std::string output_model_dtype;
     std::string granularity;
-    std::string input_type;
-    std::string output_type;
+    std::string input_type = "uint8";
+    std::string output_type = "uint8";
```

Comment:
IMHO, these two lines should be split to another PR

Reply:
Q) why `output_model_dtype` and `granularity` not given default values?

Reply:
> Q) why `output_model_dtype` and `granularity` not given default values?

Changing `input_type` and `output_type` values was needed for tests, and `granularity` and `output_model_dtype` is hardcoded now in `master`, so setting `granularity` and `output_model_dtype` is not necessary.
I'll move these changes to next PR. Thank you.

Reply:
> IMHO, these two lines should be split to another PR

done